buildvertical.targets

```diff
@@ -138,16 +142,34 @@
     <PropertyGroup>
       <_SelectedTargetGroup>%(VerticalTargets.TargetGroup)</_SelectedTargetGroup>
       <_SelectedOSGroup>%(VerticalTargets.OSGroup)</_SelectedOSGroup>
+      <_Configuration>$(_SelectedTargetGroup)</_Configuration>
+      <_Configuration Condition="'$(_SelectedOSGroup)' != '' and '$(_SelectedOSGroup)' != 'AnyOS'">$(_Configuration)-$(_SelectedOSGroup)</_Configuration>
+      <_AdditionalProperties Condition="'$(_SelectedTargetGroup)' != ''">TargetGroup=$(_SelectedTargetGroup)</_AdditionalProperties>
+      <_AdditionalProperties Condition="'$(_SelectedOSGroup)' != '' and '$(_SelectedOSGroup)' != 'AnyOS'">OSGroup=$(_SelectedOSGroup);$(_AdditionalProperties)</_AdditionalProperties>
+      <_AdditionalProperties Condition="'$(_Configuration)' != ''">Configuration=$(_Configuration);$(_AdditionalProperties)</_AdditionalProperties>
```

Comment:
I think we should try and make it such that the only property we pass through is the Configuration property (and perhaps the archgroup if it is needed later). I want to try and avoid needing to know about all these individual properties here. They are easily parsed on the project side from the Configuration.